llvm/docs/GettingStartedVS.rst

```diff
@@ -119,7 +120,43 @@ These instruction were tested with Visual Studio 2019 and Python 3.9.6:
  Select the last link: ``Source code (zip)`` and unpack the downloaded file using
  Windows Explorer built-in zip support or any other unzip tool.
 
-12. Finally, configure LLVM using CMake:
+12. Open the LLVM folder in Visual Studio or configure LLVM with CMake.
+
+    Recently Visual Studio has added native support for CMake which will run and
```

Comment:
I would suggest replacing "recently" with e.g. a version number, to make the statement more time-resistant.